pkg/platform/api/inventory/inventory.go

```diff
@@ -22,24 +23,28 @@ var persist inventory_operations.ClientService
 var transport http.RoundTripper
 
 // Init will create a new API client using default settings
-func Init() (inventory_operations.ClientService, runtime.ClientTransport) {
-	return New(api.GetServiceURL(api.ServiceInventory))
+func Init(auth *authentication.Auth) (inventory_operations.ClientService, runtime.ClientTransport) {
+	return New(api.GetServiceURL(api.ServiceInventory), auth.ClientAuth())
 }
 
 // New initializes a new api client
-func New(serviceURL *url.URL) (inventory_operations.ClientService, runtime.ClientTransport) {
+func New(serviceURL *url.URL, auth runtime.ClientAuthInfoWriter) (inventory_operations.ClientService, runtime.ClientTransport) {
 	transportRuntime := httptransport.New(serviceURL.Host, serviceURL.Path, []string{serviceURL.Scheme})
 	transportRuntime.Transport = api.NewRoundTripper()
 
 	// transportRuntime.SetDebug(true)
 
+	if auth != nil {
+		transportRuntime.DefaultAuthentication = auth
+	}
+
 	return inventory_operations.New(transportRuntime, strfmt.Default), transportRuntime
 }
 
 // Get returns a cached version of the default api client
-func Get() inventory_operations.ClientService {
+func Get(auth *authentication.Auth) inventory_operations.ClientService {
```

Comment:
This one doesn't really make sense, especially when all your use has you calling `authentication.Get()`. We know this is dated and needs to be removed, let's leave `Get()` calls alone for this PR.